lib/templates/json_texture.template.js

```diff
@@ -0,0 +1,43 @@
+path = require("path");
+
+function jsonTextureTemplate(data) {
+  var spriteObj = {};
+
+  // Create frame data for each sprite.
+  spriteObj.frames={};
+
+  data.sprites.forEach(function (sprite) {
+    var entry={
+      frame: {
+        "x": sprite.x,
+        "y": sprite.y,
+        "w": sprite.width,
+        "h": sprite.height
+      }
+    };
+
+    spriteObj.frames[path.basename(sprite.source_image)]=entry;
+  });
+
+  // Create the meta data.
+  spriteObj.meta = {
+    "app": "spritesheet-templates",
+    "version": "9.5.0",
```

Comment:
Please use single quote for object values. It is enforced by the linter and gives this repo consistency.

```js
  app: 'spritesheet-templates',
  version: '9.5.0',
```